.circleci/config.yml

```diff
@@ -168,6 +181,7 @@ jobs:
     <<: *container_config
     steps:
       - checkout
+      - *merge_checkout
```

Comment:
I see no merge_checkout on osx_package job, on the assumption that that job is not run in PR.
I see that linux_package is run both in PR and master/tags, on the assumption that CIRCLE_PR_NUMBER is non-empty string. That is reasonable.